src/ducks/categories/styles.styl

```diff
@@ -32,6 +59,22 @@
         .bnk-table-percentage
             font-weight bold
 
+    .bnk-table-row-caption
+        margin-top 0
+
+    .bnk-table-percentage-caption
+        color charcoalGrey
+
+        &::after
+            content ''
+            display inline-block
+            height 1px
+            width 1px
+            background-color coolGrey
+            vertical-align middle
+            margin-left 5px
+            margin-right 4px
```

Comment:
wha'ts this ?

Reply:
There's a veeeeery little dot between the percentage and the number of transactions : 

![image](https://user-images.githubusercontent.com/1606068/40127546-e9b5b72c-592f-11e8-9cc3-0cc5ba3c9231.png)

It's not dust on your screen, it's really a dot between elements. Don't try to remove it with your finger like I did :smile:

Reply:
lol. I think https://fr.wikipedia.org/wiki/Point_m%C3%A9dian is what needs to be used ;)